leaks.go

```diff
@@ -29,6 +29,7 @@ import (
 // TestingT is the minimal subset of testing.TB that we use.
 type TestingT interface {
 	Error(...interface{})
+	Helper()
```

Comment:
Changing the interface is a breaking change—even though all supported versions of Go have this method now.

To do this in a backwards compatible way, we have to do something like this:

```
type testHelper interface{ Helper() }

func VerifyNone(t TestingT, ...) {
  if h, ok := t.(testHelper); ok {
    h.Helper()
  }
  // ...
}
```

Reply:
That makes sense, will do.